package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "aug": "^2.0.0",
     "hapi-favicon": "^2.0.0",
-    "hapi-req": "^2.0.0",
+    "hapi-req": "2.0.0",
```

Comment:
why did this change?

Reply:
I changed it because the `^2.0.0` is installing 2.1.0 which in this case breaks the tests. Let me figure out why.

Reply:
any update on this?